packages/transform-module/test/test-static-module-record.js

```diff
@@ -0,0 +1,569 @@
+// @ts-check
+/// <reference types="ses"/>
+/* eslint no-underscore-dangle: ["off"] */
+
+import 'ses';
+import test from 'ava';
+import { StaticModuleRecord } from '../src/static-module-record.js';
+
+/** @typedef {import('ava').ExecutionContext} TestContext */
+
+/**
+ * @callback Updater
+ * @param {any} value
+ */
+/** @typedef {Map<string, Map<string, Array<Updater>>>} ImportUpdaters */
+
+/**
+ * @param {TestContext} t
+ * @param {StaticModuleRecord} record
+ */
+function assertDefaultExport(t, record) {
+  t.deepEqual(record.imports, []);
+  t.deepEqual(record.exports, ['default']);
+  t.deepEqual(record.reexports, []);
+  t.deepEqual(record.__fixedExportMap__, { default: ['default'] });
+  t.deepEqual(record.__liveExportMap__, {});
+}
+
+test('export default', t => {
+  t.plan(8);
+  const record = new StaticModuleRecord('export default bb;');
+  assertDefaultExport(t, record);
+
+  const compartment = new Compartment({
+    bb: 'bingbang',
+  });
+  const functor = compartment.evaluate(record.__syncModuleProgram__);
+  t.is(typeof functor, 'function');
+
+  functor({
+    imports: () => {
+      t.is(1, 1);
```

Comment:
I'd prefer the following idiom (untested):

```suggestion
      t.assert(true);
```